src/asim/configs/resident/summarize_preprocessor.csv

```diff
@@ -0,0 +1,434 @@
+,_walk_loc_wlk_acc,trip_odt_skims['WALK_LOC_ACC']
+,_walk_mix_wlk_acc,trip_odt_skims['WALK_MIX_ACC']
+,_walk_prm_wlk_acc,trip_odt_skims['WALK_PRM_ACC']
+,_knrin_loc_wlk_egr,trip_odt_skims['KNRIN_LOC_EGR']
+,_knrin_mix_wlk_egr,trip_odt_skims['KNRIN_MIX_EGR']
+,_knrin_prm_wlk_egr,trip_odt_skims['KNRIN_PRM_EGR']
+,_pnrin_loc_wlk_egr,trip_odt_skims['PNRIN_LOC_EGR']
+,_pnrin_mix_wlk_egr,trip_odt_skims['PNRIN_MIX_EGR']
+,_pnrin_prm_wlk_egr,trip_odt_skims['PNRIN_PRM_EGR']
+,_tncin_loc_wlk_egr,trip_odt_skims['TNCIN_LOC_EGR']
+,_tncin_mix_wlk_egr,trip_odt_skims['TNCIN_MIX_EGR']
+,_tncin_prm_wlk_egr,trip_odt_skims['TNCIN_PRM_EGR']
+,_knrout_loc_wlk_egr,trip_odt_skims['KNROUT_LOC_EGR']
+,_knrout_mix_wlk_egr,trip_odt_skims['KNROUT_MIX_EGR']
+,_knrout_prm_wlk_egr,trip_odt_skims['KNROUT_PRM_EGR']
+,_pnrout_loc_wlk_egr,trip_odt_skims['PNROUT_LOC_EGR']
+,_pnrout_mix_wlk_egr,trip_odt_skims['PNROUT_MIX_EGR']
+,_pnrout_prm_wlk_egr,trip_odt_skims['PNROUT_PRM_EGR']
+,_tncout_loc_wlk_egr,trip_odt_skims['TNCOUT_LOC_EGR']
+,_tncout_mix_wlk_egr,trip_odt_skims['TNCOUT_MIX_EGR']
+,_tncout_prm_wlk_egr,trip_odt_skims['TNCOUT_PRM_EGR']
+,_walk_loc_wlk_egr,trip_odt_skims['WALK_LOC_EGR']
+,_walk_mix_wlk_egr,trip_odt_skims['WALK_MIX_EGR']
+,_walk_prm_wlk_egr,trip_odt_skims['WALK_PRM_EGR']
+
+## Costs
+# Private auto tolls
+# Bridge
+#####,_sov_btoll,trip_odt_skims['SOV_BTOLL']
+#####,_hov2_btoll,trip_odt_skims['HOV2_BTOLL']
+#####,_hov3_btoll,trip_odt_skims['HOV3_BTOLL']
+#####,_sovtoll_btoll,trip_odt_skims['SOVTOLL_BTOLL']
+#####,_hov2toll_btoll,trip_odt_skims['HOV2TOLL_BTOLL']
+#####,_hov3toll_btoll,trip_odt_skims['HOV3TOLL_BTOLL']
+# Value
+#####,_sovtoll_vtoll,trip_odt_skims['SOVTOLL_VTOLL']
+#####,_hov2toll_vtoll,trip_odt_skims['HOV2TOLL_VTOLL']
+#####,_hov3toll_vtoll,trip_odt_skims['HOV3TOLL_VTOLL']
+,_sov_tr_l_vtoll,trip_odt_skims['SOV_TR_L_TOLLCOST']
+,_sov_tr_m_vtoll,trip_odt_skims['SOV_TR_M_TOLLCOST']
+,_sov_tr_h_vtoll,trip_odt_skims['SOV_TR_H_TOLLCOST']
+,_sov_nt_l_vtoll,trip_odt_skims['SOV_NT_L_TOLLCOST']
+,_sov_nt_m_vtoll,trip_odt_skims['SOV_NT_M_TOLLCOST']
+,_sov_nt_h_vtoll,trip_odt_skims['SOV_NT_H_TOLLCOST']
+,_hov2_l_vtoll,trip_odt_skims['HOV2_L_TOLLCOST']
+,_hov2_m_vtoll,trip_odt_skims['HOV2_M_TOLLCOST']
+,_hov2_h_vtoll,trip_odt_skims['HOV2_H_TOLLCOST']
+,_hov3_l_vtoll,trip_odt_skims['HOV3_L_TOLLCOST']
+,_hov3_m_vtoll,trip_odt_skims['HOV3_M_TOLLCOST']
+,_hov3_h_vtoll,trip_odt_skims['HOV3_H_TOLLCOST']
+
+# Transit fares
+,_knrin_loc_wlk_far,trip_odt_skims['KNRIN_LOC_FARE']
+,_knrin_mix_wlk_far,trip_odt_skims['KNRIN_MIX_FARE']
+,_knrin_prm_wlk_far,trip_odt_skims['KNRIN_PRM_FARE']
+,_pnrin_loc_wlk_far,trip_odt_skims['PNRIN_LOC_FARE']
+,_pnrin_mix_wlk_far,trip_odt_skims['PNRIN_MIX_FARE']
+,_pnrin_prm_wlk_far,trip_odt_skims['PNRIN_PRM_FARE']
+###### how are costs of TNC handled? skims are not clear
+,_tncin_loc_wlk_far,trip_odt_skims['TNCIN_LOC_FARE']
+,_tncin_mix_wlk_far,trip_odt_skims['TNCIN_MIX_FARE']
+,_tncin_prm_wlk_far,trip_odt_skims['TNCIN_PRM_FARE']
```

Comment:
There is no 'TNCIN_LOC_FARE', 'TNCIN_MIX_FARE', 'TNCIN_PRM_FARE' in transit skims